lib/studies/reply/adaptive_nav.dart

```diff
@@ -1219,7 +1220,7 @@ class _ReplyFabState extends State<_ReplyFab>
               return Tooltip(
                 message: tooltip,
                 child: InkWell(
-                  key: const ValueKey('ReplyFab'),
+                  key: K.createMailButton,
```

Comment:
Let's preserve values of the keys that was here before, like you did with `studyDemoList`.